### PR TITLE
Enhancement/style overhaul

### PR DESCRIPTION
**Pull request by acheung-94**

### CODE REVIEW

File: frontend/index.html
Comment (ChristopherJTrent) [inline frontend/index.html]: the link tags aren't necessary when using an import at-rule in your CSS
```suggestion
```

**Review by ChristopherJTrent — CHANGES_REQUESTED** [review]
One requested change, otherwise LGTM

**Review by ChristopherJTrent — APPROVED** [review]
LGTM, just waiting on review from Sam

**Review by sam-kim99 — APPROVED** [review]

**Comment by ChristopherJTrent** [comment]
:tada:

**Pull request merged**
